src/vlogger/vlogger.h

```diff
@@ -142,25 +142,27 @@ extern "C" {
 #endif //__cplusplus
 
 typedef enum {
-	VLOG_NONE  = -1,
-	VLOG_PANIC =  0,
+	VLOG_INIT = -2,
+	VLOG_NONE,
```

Comment:
I think it will be easier to understand if you specify all the negative values up to (including) the zero '0'
__INTI=-2
_NONE=-1
_PANIC=0

since it is less common to see enum start counter from -2